cmd/av/stack_sync.go

```diff
@@ -236,8 +237,35 @@ func (vm *stackSyncViewModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		vm.pruningBranches = false
 		return vm, tea.Quit
 
+	case promptUserShouldSyncAllMsg:
+		vm.syncAllPrompt = uiutils.NewPromptModel("You are on the trunk, do you want to sync all stacks?", []string{"Yes", "No"})
```

Comment:
Hey @draftcode, I would appreciate your input and guidance on the copy for this :)